src/app/systelab-components/spinner/spinner.component.ts

```diff
@@ -8,7 +8,7 @@ import { TouchSpinValues } from './touch.spin-values';
 })
 export class TouchspinComponent {
 
-	public isDisabled = false;
+	@Input() public isDisabled = false;
```

Comment:
It should be consistent across components. There are 3 of them with "disabled" (switch, datepicker and toggle) and 4 with "isDisabled" (combo, searcher, month and week selector).